src/migrations/2016_08_03_072729_create_provinces_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create(config('laravolt.indonesia.table_prefix') . 'provinces', function(Blueprint $table)
         {
-            $table->char('id', 2);
+            $table->char('id', 3);
```

Comment:
Why you change this?

Reply:
I've spec add more Province data. and it reach more than 99. so I modify that for support the spec.

Reply:
Please make separate PR for those

Reply:
ok cool @uyab!